src/bond.py

```diff
@@ -63,3 +63,104 @@ def __init__(self):
             ),
         )
         self._add_typeparam(params)
+
+
+class Quartic(bond.Bond):
+    r"""Quartic bond potential.
+
+    `Quartic` specifies a quartic potential between the two particles in
+    the simulation state with:
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        U(r) &= k (r - \Delta - r_0 - b_1)(r - \Delta - r_0 - b_2)(r - \Delta - r_0)^2
+                  + U_0 + U_{\rm WCA}(r), & r < r_0 + \Delta \\
+             &= U_0, & r \ge r_0 + \Delta
+        \end{eqnarray*}
```

Comment:
This equation is displaying poorly because it's too wide. Please try to reformat it in a way that looks nicer. You may need to build the docs locally to figure out what works best.

Reply:
Sorry about the delay! Took some time to figure out how to build the docs locally. Thank you so much for catching this, as well as all the other feedback!

Reply:
No problem! Let me know when you're ready for another look.